boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts

```diff
@@ -15,6 +15,20 @@
 	compatible = "infineon,xm4500-relax-kit", "infineon,xmc4500",
 						"infineon,xmc4xxx";
 
+	aliases {
+		led0 = &led1;
```

Comment:
???

Reply:
It's an alias for running the blinky sample:
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/basic/blinky/src/main.c#L14

The leds are labeled led1 and led2 in the relax kit documentation:
https://www.infineon.com/dgdl/Board_Users_Manual_XMC4500_Relax_Kit-V1_R1.2_released.pdf?fileId=db3a30433acf32c9013adf6b97b112f9

Reply:
This at least deserves some source code comments IMO.

Reply:
will update.

Reply:
Thanks! I agree this is resolved.